staff/ramon-palao/precourse/Calculadora HTML/index.js

```diff
@@ -0,0 +1,163 @@
+//Declaro los eventos para cada una de las teclas.
+document.getElementById("reset").addEventListener("click", acFunc);
+document.getElementById("borrar").addEventListener("click", borrFunc);
+document.getElementById("division").addEventListener("click", divisFunc);
+document.getElementById("seven").addEventListener("click", num7);
+document.getElementById("eight").addEventListener("click", num8);
+document.getElementById("nine").addEventListener("click", num9);
+document.getElementById("multi").addEventListener("click", multiFunc);
+document.getElementById("four").addEventListener("click", num4);
+document.getElementById("five").addEventListener("click", num5);
+document.getElementById("six").addEventListener("click", num6);
+document.getElementById("menos").addEventListener("click", menosFunc);
+document.getElementById("one").addEventListener("click", num1);
+document.getElementById("two").addEventListener("click", num2);
+document.getElementById("three").addEventListener("click", num3);
+document.getElementById("mas").addEventListener("click", masFunc);
+document.getElementById("zero").addEventListener("click", num0);
+document.getElementById("coma").addEventListener("click", comaFunc);
+document.getElementById("igual").addEventListener("click", igualFunc);
+
+//Declaro cada una de las funciones incluidas en los eventos
+//Para resetear toda la pantalla declaro la siguiente función:
+function acFunc(){
+    var reset = [];
+    document.getElementById("pantalla").innerHTML = reset;
+}
+
+//Las siguientes funciones permiten que en la pantalla de la calculadora se muestren los signos o números que tecleemos.
+function divisFunc(){
+    var display = document.getElementById("pantalla").innerHTML;
```

Comment:
You can remove .innerHTML and always the line 30 works

Reply:
Hi Victori, thanks for your comment!
I have tried removing .innerHTML from line 30, but if I do that the calculator screen does not print the mathematical operation and next message is shown: [object HTMLDivElement]÷